### DISCUSSION TITLE
Bot still doesn't load tracks after the update

### DISCUSSION THREAD

**Discussion by MisterSheeple** [jagrosh/MusicBot#1308] [Troubleshooting]
Not sure why this is happening.

**Comment by MichailiK** [comment]
Does it disconnect & reconnect the voice channel for a quick second? Try changing your voice regions, it might be that Discord's change hasn't entirely rolled out to all regions yet. I somewhat doubt that's the case, but you can try changing your voice region in the channel settings & see if that fixes it.

**Reply by MisterSheeple** [reply]
That didn't fix it.
